data.json

```diff
@@ -81758,6 +81751,13 @@
       "label": "antidot",
       "language": "php",
       "website": "https://github.com/antidot-framework/react-framework"
+    },
+    {
+      "id": 253,
+      "version": 4.1,
```

Comment:
Indeed @josephmancuso, `masonite` use SEMVER. But having a constraint `>=4,<5` will not be in-sync with version

I mean having version for each framework, add accuracy for this project, imho

Reply:
True it needs to be equal to this number I see